### PR TITLE
Fixed the media component issues

### PR DESCRIPTION
**Pull request by IbrahimSulai**
Fixed the below issues in media component:
1. Handled the empty media URL loads in order to avoid the crash.
2. Updated the "paused" attribute value to true - To avoid confusion on auto-playing more than one media file on the launch.

### CODE REVIEW

File: source/community/reactnative/src/components/elements/media.js [hunk 1]
```diff
@@ -87,13 +91,13 @@ export class Media extends React.Component {
...
                                     <Video
                                         source={this.sources[this.state.currentSourceIndex]}
                                         fullscreen={true}
                                         controls={true}
                                         id={this.payload.id ? this.payload.id : "video"}
-                                        paused={false}
+                                        paused={true}
                                         onError={() => { this.videoError(onParseError) }}
                                         onLoad={this.videoLoadSuccess}
                                         style={styles.nativeVideoControls}
```
Comment (BalajiR) [inline source/community/reactnative/src/components/elements/media.js]: Poster displayed is not showing the full image.

File: source/community/reactnative/src/components/elements/media.js [hunk 2]
```diff
@@ -87,13 +91,13 @@ export class Media extends React.Component {
                         <ElementWrapper json={this.payload} isFirst={this.props.isFirst}>
```
Comment (BalajiR) [inline source/community/reactnative/src/components/elements/media.js]: Remove the InputContextConsumer and use context via static prop.

File: source/community/reactnative/src/components/elements/media.js [hunk 3]
```diff
@@ -39,7 +39,7 @@ export class Media extends React.Component {
                 this.context.addResourceInformation(source.url, source.mimeType);
             })
         }
```
Comment (BalajiR) [inline source/community/reactnative/src/components/elements/media.js]: Add description for every custom method.

File: source/community/reactnative/src/components/elements/media.js [hunk 4]
```diff
@@ -48,6 +48,10 @@ export class Media extends React.Component {
         if (this.payload.sources && this.payload.sources.length > 0) {
             sources.forEach(source => {
                 this.addUriAttribute(source);
+                //removing the source if its uri is empty
+                if (Utils.isNullOrEmpty(source.uri)) {
```
[the remark this answers is not in the document]
Comment (IbrahimSulai) [reply source/community/reactnative/src/components/elements/media.js]: Trimmed the empty white spaces. Resolved

File: source/community/reactnative/src/components/elements/media.js (hunk 1 above)
Comment (IbrahimSulai) [reply source/community/reactnative/src/components/elements/media.js]: Added the resize mode for the poster image. Resolved

File: source/community/reactnative/src/components/elements/media.js (hunk 2 above)
Comment (IbrahimSulai) [reply source/community/reactnative/src/components/elements/media.js]: Resolved

File: source/community/reactnative/src/components/elements/media.js (hunk 3 above)
Comment (IbrahimSulai) [reply source/community/reactnative/src/components/elements/media.js]: Added Description. Resolved

File: source/community/reactnative/src/components/elements/media.js [hunk 5]
```diff
@@ -39,29 +39,48 @@ export class Media extends React.Component {
...
+    /**
+     * @description Add the uri attribute to the source
+     * @param {object} source - source from the sources array
+     * @returns {object} source with trimmed uri attribute
+     */
     addUriAttribute = (source) => {
         if (source.url) {
             source.uri = source.url
         }
+        source.uri = source.uri.trim()
```
Comment (BalajiR) [inline source/community/reactnative/src/components/elements/media.js]: Either move the `trim` on `url` OR initialize uri with an empty string if the url is absent.

Comment (IbrahimSulai) [reply source/community/reactnative/src/components/elements/media.js]: Added conditional check.

**Pull request merged**